### ISSUE TITLE
Package path ./lab/overwriteReadonlyProp is not exported from package node_modules/tsafe

### ISSUE THREAD

**Issue by thomasraydeniscool** [garronej/evt#50]
When I try to use Evt with `vue@2.7` and `@vue/cli-service@5.0.8` I get the errors below.

I'm not too sure what the issue is so any help would be greatly appreciated. I did find this issue which I think is related https://github.com/vuejs/vue-cli/issues/7144.

```
ERROR Failed to compile with 1 error
error in ./node_modules/evt/lib/Ctx.js

Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)

ERROR in ./node_modules/evt/lib/Ctx.js 33:30-72
Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)
 @ ./node_modules/evt/lib/index.js 20:12-28
 @ ./src/services/active-org-service.ts 15:0-26 118:31-34
 @ ./src/App.ts 35:0-65 146:21-51
 @ ./src/main.ts 18:0-24 45:13-16

ERROR in ./node_modules/evt/lib/Evt.js 126:30-72
Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)
 @ ./node_modules/evt/lib/index.js 22:12-28
 @ ./src/services/active-org-service.ts 15:0-26 118:31-34
 @ ./src/App.ts 35:0-65 146:21-51
 @ ./src/main.ts 18:0-24 45:13-16

ERROR in ./node_modules/evt/lib/LazyEvt.js 4:30-72
Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)
 @ ./node_modules/evt/lib/Ctx.js 31:16-36
 @ ./node_modules/evt/lib/index.js 20:12-28
 @ ./src/services/active-org-service.ts 15:0-26 118:31-34
 @ ./src/App.ts 35:0-65 146:21-51
 @ ./src/main.ts 18:0-24 45:13-16

ERROR in ./node_modules/evt/lib/LazyStatefulEvt.js 4:30-72
Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)
 @ ./node_modules/evt/lib/StatefulEvt.js 46:24-52
 @ ./node_modules/evt/lib/index.js 24:20-44
 @ ./src/services/active-org-service.ts 15:0-26 118:31-34
 @ ./src/App.ts 35:0-65 146:21-51
 @ ./src/main.ts 18:0-24 45:13-16

ERROR in ./node_modules/evt/tools/Deferred.js 19:30-72
Module not found: Error: Package path ./lab/overwriteReadonlyProp is not exported from package /home/office-app/node_modules/tsafe (see exports field in /home/office-app/node_modules/tsafe/package.json)
 @ ./node_modules/evt/lib/Evt.js 128:17-45
 @ ./node_modules/evt/lib/index.js 22:12-28
 @ ./src/services/active-org-service.ts 15:0-26 118:31-34
 @ ./src/App.ts 35:0-65 146:21-51
 @ ./src/main.ts 18:0-24 45:13-16

webpack compiled with 5 errors
No issues found.
```

**Comment by garronej** [comment]
Thanks for reporting, very sorry about that!

**Comment by garronej** [comment]
It's fixed in EVT v2.4.13.  
Again sorry for the inconvegnence, it won't happen again.

**Comment by thomasraydeniscool** [comment]
Worked like a charm, thanks!
